### DISCUSSION TITLE
Feature request: Deduplicate the contents of only some directories.

### DISCUSSION THREAD

**Discussion by virtadpt** [linuxmint/discussions#721] [Ideas]
Hi.  I'm using Pix to deduplicate my assets collection, and I realized something: For Pix to build a file list (which is recursive by default) and then analyze all of the files it finds for similarity can take a long time indeed (sometimes hours at a time).  Deduplicating a single directory non-recursively, however, is comparatively simple and quick.

What if it was possible to select a base directory (assets/) in the file browser pane (which implicitly expands all of the subdirectories inside it), then select a few of those subdirectories (assets/avatars/, assets/icons/, assets/tiles/, assets/emojo/) and set off deduplication?  To put it another way, I'd like to deduplicate the base directory along with a handful of subdirectories at a time so that the process doesn't necessarily have to take three days straight.

Would this be of interest to other users?

**Comment by ChooChooAl** [comment]
Would this work?
fdupes -r .  
will give you a list,  granted it's done on the command line. 
fdupes -r . >mylist.txt
would save it to a file and you could then scroll through it with you text editor.
